client/src/components/Layout/GamePage.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
Don't disable this eslint rule. This is important. Nested ternary is making it difficult to understand code.

Reply:
@kaushikb11 Why didn't you remove it?